lib/openstudio-standards/standards/Standards.PlantLoop.rb

```diff
@@ -55,7 +55,8 @@ def chw_sizing_control(model, chilled_water_loop, dsgn_sup_wtr_temp, dsgn_sup_wt
   # @param model [OpenStudio::Model::Model] OpenStudio model object
   # @return [String] common_pipe or heat_exchanger
   def plant_loop_set_chw_pri_sec_configuration(model)
-    pri_sec_config = 'common_pipe'
+    # pri_sec_config = 'common_pipe'
+    pri_sec_config = 'heat_exchanger'
```

Comment:
That's a fair point-- I'm not as familiar with how broadly this function is called. It looks like this function is called in Prototype.hvac_systems.rb under model_add_chw_loop and is used whenever the `pumping_type` is set to 'const_pri_var_sec'. This is called when setting a few different configurations:

- District Cooling
- [Heat Pumps?
](https://github.com/NREL/openstudio-standards/blob/87091ad8c6171b8ac6c96baa176d9da06614699c/lib/openstudio-standards/prototypes/common/objects/Prototype.hvac_systems.rb#L6255)
- Water Cooled chillers

That brings up a good question of if we'd like to restrict this methodology to certain configurations. If not-- we'd want to add a little more logic here.  I'm imagine the vision was the enumeration for the pump configuration would be called in many places using primary/secondary configurations and this would carry elsewhere. Happy to remove the comment.

Reply:
Looks like this was changed.